keras_cv/layers/preprocessing/base_image_augmentation_layer.py

```diff
@@ -57,14 +57,23 @@ class BaseImageAugmentationLayer(tf.keras.__internal__.layers.BaseRandomLayer):
     coodinate the randomness behavior, eg, in the RandomFlip layer, the image
     and bounding_boxes should be changed in the same way.
 
-    The `call()` method support two formats of inputs:
-    1. Single image tensor with 3D (HWC) or 4D (NHWC) format.
-    2. A dict of tensors with stable keys. The supported keys are:
-      `"images"`, `"labels"` and `"bounding_boxes"` at the moment. We might add
-      more keys in future when we support more types of augmentation.
-
-    The output of the `call()` will be in two formats, which will be the same
-    structure as the inputs.
+    The `call()` method supports two formats of inputs:
+    1. A single image tensor with shape (height, width, channels) or
+       (batch_size, height, width, channels)
+    1. A dict of tensors with stable keys. The supported keys are:
```

Comment:
What does "stable" mean in this context?

Reply:
I don't think this was the right word choice -- I've updated the wording a bit.